next.config.mjs

```diff
@@ -10,6 +10,8 @@ const withMDX = MDX();
 /** @type {import('next').NextConfig} */
 const nextConfig = {
 	reactStrictMode: true,
+	// uncomment the following line in case you build to docker
+	// output: "standalone",
```

Comment:
This could be controlled via an env variable, so there's no need to edit the config file. Something like:
```suggestion
	output: process.env.DOCKER ? "standalone" : undefined,
```

Reply:
I've added your suggestion